tools/config/winlogbeat-modules-enabled.yml

```diff
@@ -135,6 +135,7 @@ fieldmappings:
   Product: winlog.event_data.Product
   Properties: winlog.event_data.Properties
   RuleName: winlog.event_data.RuleName
+  ScriptBlockText: winlog.event_data.ScriptBlockText
```

Comment:
modules enabled = ECS.
should be powershell.file.script_block_text
see: https://github.com/elastic/beats/blob/048c3cc19bf43c8a6b332afaafdd0a2eb8e5bd49/x-pack/winlogbeat/module/powershell/config/winlogbeat-powershell.js#L171-L178

Reply:
Great catch! It's been updated accordingly.